### DISCUSSION TITLE
No boot ZuluSCSI RP2040 on MPC2000XL

### DISCUSSION THREAD

**Discussion by gpcpg** [ZuluSCSI/ZuluSCSI-firmware#120] [Q&A]
Hello,

I installed ZuluSCSI RP2040 to AKAI MPC2000XL.
But sometimes firmware isn't loaded, no LED blink. (MPC says "NO SCSI device")
It appears when SD card is mounted or not mounted. (I guess SD card is no problem)
Once firmware loaded correctly, I can access img files, save/load is OK.
I installed latest firmware(1.1.5) and also tried it on another MPC2000XL, but not solved. 
How can I fix it?

Thanks

**Comment by aperezbios** [comment]
Hi there, how are you powering your ZuluSCSI? It sounds like this may be a power reliability issue.

**Reply by gpcpg** [reply]
I tried SCSI bus power and 5V connector (from MPC main board), but not solved. thanks
